.github/workflows/push.yml

```diff
@@ -1,9 +1,7 @@
 name: GitHub Actions
-on: [push]
+on: [ push ]
 env:
-  # Dev docker image, must be same as in docker-compose.yml
-  IMAGE: keboolabot/keboola-as-code-dev
-  IMAGE_TAG: ${{ github.sha }}
```

Comment:
Na build sa uz viac nepouziva `Docker`, ale nativne `go`, kedze na Windows sa to cez Docker nedalo rozbehat.